addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -4404,11 +4504,15 @@ class SnippetsMenu extends Component {
     /**
      * @private
      */
-    _onSnippetClick() {
-        const $els = this.getEditableArea().find('.oe_structure.oe_empty').addBack('.oe_structure.oe_empty');
-        for (const el of $els) {
-            if (!el.children.length) {
-                $(el).odooBounce('o_we_snippet_area_animation');
+    _onSnippetClick(currentTarget) {
```

Comment:
cf. comment in `snippets.xml` => it's a handler, I think using `ev` as a param would make more sense